backend/api/src/main/java/com/virtuslab/gitmachete/backend/api/NullBranchLayout.java

```diff
@@ -44,4 +44,9 @@ public IBranchLayout slideIn(String parentBranchName, IBranchLayoutEntry entryTo
   public IBranchLayout slideOut(String branchName) {
     return this;
   }
+
+  @Override
+  public List<String> getOrderedEntriesNames() {
```

Comment:
```suggestion
  public List<String> getOrderedEntryNames() {
```
https://virtuslab.github.io/tips/#english/grammar-tips-1 -> point 4